src/server/server_family.h

```diff
@@ -178,6 +183,8 @@ class ServerFamily {
   std::unique_ptr<ScriptMgr> script_mgr_;
   std::unique_ptr<journal::Journal> journal_;
   std::unique_ptr<DflyCmd> dfly_cmd_;
+  std::unique_ptr<ChannelStore> channel_store_;
```

Comment:
nit: it could also be in main_service i guess

Reply:
Where should it be? It just seemed that server family stores all command specific stuff (like all the things above), and main_service stores management things